org.gimp.BaseApp.json

```diff
@@ -0,0 +1,279 @@
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "http://ftp.gnome.org/pub/GNOME/sources/pygobject/2.28/pygobject-2.28.6.tar.xz",
+                            "sha256": "fb8a1d4f665130a125011659bd347c7339c944232163dbb9a34fd0686577adb8"
+                        }
+                    ]
+                },
+                {
+                    "name": "pygtk",
+                    "cleanup": [ "/lib/pygtk" ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "http://ftp.gnome.org/pub/GNOME/sources/pygtk/2.24/pygtk-2.24.0.tar.gz",
+                            "sha256": "6e3e54fa6e65a69ac60bd58cb2e60a57f3346ac52efe995f3d10b6c38c972fd8"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "gexiv2",
+            "modules": [
+                {
+                    "name": "exiv2",
+                    "cleanup": [ "/bin" ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "http://www.exiv2.org/exiv2-0.25.tar.gz",
+                            "sha256": "c80bfc778a15fdb06f71265db2c3d49d8493c382e516cb99b8c9f9cbde36efa4"
+                        }
+                    ]
+                }
+            ],
+            "config-opts": [ "--disable-introspection" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gexiv2/0.10/gexiv2-0.10.6.tar.xz",
+                    "sha256": "132788919667254b42c026ab39ab3c3a5be59be8575c05fa4b371ca8aed55825"
+                }
+            ]
+        },
+        {
+            "name": "openexr",
+            "modules": [
+                {
+                    "name": "ilmbase",
+                    "config-opts": [ "--disable-static" ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "http://download.savannah.nongnu.org/releases/openexr/ilmbase-2.2.0.tar.gz",
+                            "sha256": "ecf815b60695555c1fbc73679e84c7c9902f4e8faa6e8000d2f905b8b86cedc7"
+                        }
+                    ]
+                }
+            ],
+            "config-opts": [ "--disable-static", "--disable-ilmbasetest" ],
+            "cleanup": [ "/bin", "/share/doc" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.savannah.nongnu.org/releases/openexr/openexr-2.2.0.tar.gz",
+                    "sha256": "36a012f6c43213f840ce29a8b182700f6cf6b214bea0d5735594136b44914231"
+                }
+            ]
+        },
+        {
+            "name": "jasper",
+            "cmake": true,
```

Comment:
`"buildsystem": "cmake-ninja"`

Reply:
Ok added. Untested, I just added the exact property as you said.